libraries/AdaptiveExpressions/Memory/ReflectionMemory.cs

```diff
@@ -27,7 +27,7 @@ public void SetValue(string path, object value)
             this.methods.SetValue.Invoke(obj, new object[] { value });
         }
 
-        public bool TryGetValue(string path, out object value)
+        public bool TryGetValue(string path, out object value, bool allowSubstitution)
```

Comment:
>allowSubstitution [](start = 68, length = 17)

Won't this complain about value not being used?

Reply:
no it won't, and it will if i call it "_", if will trigger a rule that parameter start with lower case letter.